pom.xml

```diff
@@ -517,12 +521,12 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <raml-module-builder.version>31.1.5</raml-module-builder.version>
-    <vertx.version>3.9.4</vertx.version>
+    <raml-module-builder.version>32.0.0</raml-module-builder.version>
```

Comment:
Might be worth going to `32.0.2` given that has now been released.

Reply:
Or RMB 32.1.0 which was also released and has more features.

Reply:
Even better, thanks @adamdickmeiss

Reply:
POM is updated.  I have also removed the Awaitility import.